### DISCUSSION TITLE
Azerbaijani language

### DISCUSSION THREAD

**Discussion by elnursamedzade** [RHVoice/RHVoice#493] [Q&A]
Hi!
I created voice and language files for the Azerbaijani language. How can I now use my language and voice with RHVoice?
I don't know how to create emojis right now. Which path should I follow for this?
Thank you.

**Comment by zstanecic** [comment]
Hi Elnur!

To compile your voice for use, you need visual studio or build tools, 

Python 3.7, scons and poedit.

If you have telegram, or whatsapp, I can contact you and assist with this

My telegram and phone number:

+385953577584

**Comment by elnursamedzade** [comment]
Hello again @zstanecic!
I speak Russian and English very badly. That's why I have to communicate in writing, not verbally. It can be telegram or whatsapp, but it will be more comfortable for me if it is gmail or facebook.

Here I leave my gmail address. Please contact me.
elnur.samedzade@gmail.com

**Comment by rasulovk** [comment]
Salam @elnursamedzade  alindi azerbaijan dil ile ? :)
